modules/effects/src/tokens.ts

```diff
@@ -3,24 +3,25 @@ import {
   defaultEffectsErrorHandler,
   EffectsErrorHandler,
 } from './effects_error_handler';
+import { FunctionalEffect } from './models';
 
 export const _ROOT_EFFECTS_GUARD = new InjectionToken<void>(
   '@ngrx/effects Internal Root Guard'
 );
-export const USER_PROVIDED_EFFECTS = new InjectionToken<Type<any>[][]>(
+export const USER_PROVIDED_EFFECTS = new InjectionToken<Type<unknown>[][]>(
   '@ngrx/effects User Provided Effects'
 );
-export const _ROOT_EFFECTS = new InjectionToken<Type<any>[]>(
-  '@ngrx/effects Internal Root Effects'
+export const _ROOT_EFFECTS = new InjectionToken<
+  [Array<Type<unknown> | Record<string, FunctionalEffect>>]
+>('@ngrx/effects Internal Root Effects');
+export const ROOT_EFFECTS_INSTANCES = new InjectionToken<unknown[]>(
```

Comment:
This token is "marked" as public, did we forget to add it to the barrel file?
Otherwise, I think we should rename this to be consistent with the other tokens.
```suggestion
export const _ROOT_EFFECTS_INSTANCES = new InjectionToken<unknown[]>(
```